README.md

```diff
@@ -116,6 +116,9 @@ With automatic detection of: Windows / MacOS / Linux / Intel / ARM.
 arkade get APP
 arkade get APP --version VERSION
 arkade get APP@VERSION
+
+# Dry Run
+arkade get APP@VERSION --dry-run
```

Comment:
This is a wider observation, which may need broader consideration.  The terminology used here is inconsistent with that used within arkade itself.  Is this a conscious decision? For example, is there a distinction between when `app` is to be used versus when `tool` is used?  If the two terms are interchangeable then I would suggest choosing one, again for the benefit of unfamiliar users.

Reply:
Yes, I second this, should be `tool` for a clearer distinction.

Reply:
Yes "app" is a Kubernetes application.

Then we have tools / CLIs for downloads.